templates/myid.j2

```diff
@@ -1 +1,9 @@
+{% if install_mode == "standalone" %}
 {{ zoo_id }}
+{% elif install_mode == "cluster" %}
+{% for server in play_hosts %}
```

Comment:
Not sure about using variable `play_hosts`. What happen when you execute that role along other roles and targeting other hosts?